sources/SessionView.m

```diff
@@ -50,7 +50,9 @@ + (double)titleHeight {
 }
 
 + (void)initialize {
-    lastResizeDate_ = [[NSDate date] retain];
+    if (self == [SessionView self]) {
```

Comment:
I gather this is to prevent potential subclassing-related problems?

I've also never seen the `+self` class method. Is it the same as `+class`?